hack/pull_kfp_upstream.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
a doc for the pull action?

Reply:
just use it `./hack/pull_kfp_upstream.sh`

Anything else we'd want to document?